packages/server/utils/makeCreateGitHubTaskComment.ts

```diff
@@ -0,0 +1,15 @@
+import {ExternalLinks} from '../../client/types/constEnums'
+
+// wrap anything that looks like an issue in ticks so it doesn't turn into a link to that issue
+const makeCreateGitHubTaskComment = (
+  creator: string,
+  assignee: string,
+  teamName: string,
+  teamDashboardUrl: string
+) =>
+  `Created by ${creator} for ${assignee}
+  See the dashboard of [${teamName}](${teamDashboardUrl})
+
+  *Powered by [Parabol](${ExternalLinks.INTEGRATIONS_JIRA})*`
```

Comment:
should be `INTEGRATIONS_GITHUB` 🕵️‍♂️

Reply:
Good catch, I especially added this constant :facepalm: